common/config/azure-pipelines/templates/gather-docs.yaml

```diff
@@ -103,6 +103,6 @@ steps:
       project: 2c48216e-e72f-48b4-a4eb-40ff1c04e8e4
       pipeline: 7436 # iTwin.js/Docs/iTwin.js Docs - YAML
       buildVersionToDownload: latestFromBranch
-      branchName: refs/heads/master
+      branchName: refs/heads/release/4.1.x
```

Comment:
Since this is the first 4.1.x release, we won't have an .updated.json. We should set this to release/4.0.x, then on the next 4.1.x release set it to 4.1.x.

Reply:
Now that 4.1.1 was released can we update this to 4.1.x?

Reply:
Yes absolutely.